angularjs-portal-home/src/main/webapp/js/app-config.js

```diff
@@ -26,7 +26,8 @@ define(['angular'], function(angular) {
         .constant('NAMES', {
             'title' : 'MyUW',
             'crest' : 'img/uwcrest_web_sm.png',
-            'crestalt' : 'UW Crest'
+            'crestalt' : 'UW Crest',
+            'sublogo' : ''
```

Comment:
Missed this subtlety previously, but may want to keep the "beta" sub-logo.  Cf. [design recommendations](https://docs.google.com/a/wisc.edu/document/d/14X1v0N0TPWecEkVbLPwAw3YDZv0LMZ3SnQEwMwXrorg/edit?usp=sharing), 

> The “beta” text on the logo will remain as an acknowledgement that this is a work in progress, buying some leniency from both users and content providers:

> When MyUW Classic is retired, the beta text will be removed from the MyUW logo.